docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,556 @@
+
+* [.NET CLI](http://dotnet.github.io/getting-started/), the tooling which will eventually replace DNX
+
+Either toolchain can be used to build and publish libraries today.
+
+The sections of this document dealing with .NET 4.5 Framework, .NET 4.0 Framework, or Portable Class Libraries (PCLs) require Windows and the .NET Framework installed.  The best way to do this is to [install Visual Studio](https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx).  Alternatively, you can [install .NET Framework 4.6](https://www.microsoft.com/en-us/download/details.aspx?id=48130) as a standalone package.  This will contain all the dependencies required for .NET Framework 4.5, 4.0, and any PCLs you may wish to support.
+
+## How to target .NET Core
+
+For the purposes of building a library, targeting ".NET Core" means targeting the .NET Standard Platform.  To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md).
+
+A distilled version is this:
+
+You need to pick a version of `dotnetXX` (where `XX` is a version number) to add to your `project.json` file.
+
+1. If you don't care about backwards compatibility, target `dotnet55`.
+2. If you care about compatibility with any of the following:
+
+    - .NET Framework versions from 4.5 and up, with Windows features
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    Then [refer to this table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) and choose the version of the `dotnetXX` moniker you need.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet55`:
+
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet55":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `dotnet51`:
+    
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet51":{}
+        }
+    }
+    ```
+3. If you care about compatibility with the .NET Framework versions 4.0 or below, or need to use .NET Framework APIs not available in .NET Core, skip to the next section.
+
+## How to target both .NET Core and .NET Framework
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+You may need a library to work on .NET Core *and* .NET Framework 4.0 or older.  You may also need to use specific libraries available in .NET 4.5 which are unavailable in .NET Core (such as `SYSTEM.FIXME.SOON`).  For these scenarios, you will need to specifically target the versions of .NET Framework you support.
```

Comment:
- You should spell out the full product name .NET Framework 4.5, instead of .NET 4.5. 
- "For these scenarios, you will need to specifically target the versions of .NET Framework you **want to** support." perhaps?
- Another suggestion: "You may need **to create** a library **that works on both** .NET Core *and*  .NET Framework **(version 4.0 or older)**."